cmd/commands/common.go

```diff
@@ -585,3 +585,13 @@ func askUserIfToProceedWithInsecure(ctx context.Context) error {
 	}
 	return nil
 }
+
+func FindIndexIfExists(slice []string, val string) (int, bool) {
```

Comment:
gs logic - and for future use

Reply:
move to pkg/util/util.go rename to `StringIndexOf`, no need to return `bool` it is a standard that those functions that return index return `-1` if not found.